app/models/rights_flow.rb

```diff
@@ -0,0 +1,84 @@
+# This model is not persisted in the database!
+class RightsFlow
+  include ActiveModel::Model
+  include ActiveModel::AttributeMethods
+
+  FIELDS = %w[
+    flag_a_assert_dda flag_a_let_me_choose
+
+    flag_b_critical_stage
+    flag_d_release_hearings
+    flag_e_revocation_hearings
+
+    flag_k_restitution
+
+    flag_h_limited_distribution
+    flag_i_no_media
+    flag_j
+    flag_f
+
+    first_name last_name email phone_number
+    case_number
+  ]
+
+  PAGES = %w[
+    who_assert
+    to_notification
+    to_financial_assistance
+    in_special_cases
+    create_account
+    confirm
+  ]
+
```

Comment:
Point taken, this is confusing indeed. A quick fix is some better documentation, I'll push another commit here that removes the unnecessary blank lines. (I was trying to group fields together based on which page they appear on, but I see why that is misleading.)

As for the use of these arrays in constants altogether, I'm not sure another approach that is as simple...